mesonbuild/interpreter.py

```diff
@@ -980,8 +981,11 @@ def first_supported_argument_method(self, args, kwargs):
         mlog.log('First supported argument:', mlog.red('None'))
         return []
 
-class ModuleState:
-    pass
+ModuleState = namedtuple('ModuleState', [
```

Comment:
This change seems unrelated to the rest of the changes. Did you add this on purpose?